File: source/repos/BlackJack/BlackJack.ViewModels/HistoryViews/GetPlayerStepsHistoryView.cs
using BlackJack.DataAccess.Enums;
using System;
using System.Collections.Generic;

namespace BlackJack.ViewModels.HistoryViews
{
    public class GetPlayerStepsHistoryView
    {
        public Guid GameId { get; set; }
        public string Name { get; set; }
        public List<CardGetPlayerStepsHistoryViewItem> Steps { get; set; }

        public GetPlayerStepsHistoryView()
        {
            Steps = new List<CardGetPlayerStepsHistoryViewItem>();
        }
    }
    public class CardGetPlayerStepsHistoryViewItem
    {
        public CardRankType Rank { get; set; }
        public CardSuitType Suit { get; set; }
    }
}

File: source/repos/BlackJack/BlackJack.WEB/ClientApp/src/app/shared/entities/history/get-player-steps-history.view.ts
import { CardRankType } from 'src/app/shared/enums/card-rank-type.view';
import { CardSuitType } from 'src/app/shared/enums/card-suit-type.view';

export class GetPlayerStepsHistoryView{
    gameId:string;
    name:string;
    steps:CardGetPlayerStepsHistoryViewItem[] = [];
}
export class CardGetPlayerStepsHistoryViewItem{
    rank:CardRankType;
    suit:CardSuitType;
}